### PR TITLE
Sanity check

### PR DESCRIPTION
**Pull request by jnelson16**

### CODE REVIEW

File: quantgov/corpora/builtins.py [hunk 1]
```diff
@@ -305,3 +307,91 @@ def process_document(doc, backend, precision):
...
+    @staticmethod
+    def find_extreme_documents(args):
+        df = pd.read_csv(args['metadata'])
+        results = []
+        results.append("EXTREME DOCUMENTS")
+        # The following code finds the max_words, min_words,
+        # and the locations of those documents.
+        max_words_doc = df[df['words'] == np.max(df.words)]\
+            .iloc[:, 0:SanityCheck.find_last_idx(df)].values.tolist()[0]
+        max_words_doc = 'data/clean/' + \
```
Comment (OliverSherouse) [inline quantgov/corpora/builtins.py]: This can't be hardcoded.

File: quantgov/corpora/builtins.py [hunk 2]
```diff
@@ -305,3 +307,91 @@ def process_document(doc, backend, precision):
...
+            '/'.join([str(x) for x in max_words_doc]) + '.txt'
+        results.append(('Largest document by wordcount: ' +
+                        '{} words, '.format(str(np.max(df.words))) +
+                        'in file {}'.format(max_words_doc)))
+
+        min_words_doc = df[df['words'] == np.min(df.words)]\
+            .iloc[:, 0:SanityCheck.find_last_idx(df)].values.tolist()[0]
+        min_words_doc = 'data/clean/' + \
```
Comment (OliverSherouse) [inline quantgov/corpora/builtins.py]: Likewise.

**Comment by jnelson16** [comment]
@OliverSherouse ready for another look

**Comment by jnelson16** [comment]
@OliverSherouse Ready for round 3

**Pull request closed without merging**